.github/pull_request_template.md

```diff
@@ -1,40 +1,26 @@
 ## Overview
 
 <!--
+What GitHub Desktop issue does this PR address? (for example, #1234)
+-->
 
-What issue are you addressing? (for example, #1234)
+Closes #[issue number]
 
-If an issue doesn't exist for this pull request (PR) to address, please open one
-to allow for discussion before opening this PR.
+### Screenshots
 
-You can open a new issue at https://github.com/desktop/desktop/issues/new/choose
+<!--
+If this PR touches the UI layer of the app, please include screenshots or animated gifs to show the changes.
 -->
 
-**Closes #{issue number}**
-
 ## Description
```

Comment:
What if "Description" was above "Screenshots"? So the layout is:

Closes issue x

Description
Screenshots
Release notes

Not wedded to this, just feels a bit more straightforward to me.

Reply:
oh yeah i like that better! that... might have been what i meant to do!

Reply:
done in https://github.com/desktop/desktop/pull/8391/commits/d38f2be855e58af65f9e20911a8c930f4e1a1a33